docs/user-guide/widgets.md

```diff
@@ -47,36 +46,59 @@ From the toolbar of this panel <img src="../img/widgets/widget-options.jpg" clas
 
 * Move forward <img src="../img/button/next.jpg" class="ms-docbutton"/> to the next step when the settings are completed
 
-Just below the chart's preview, the following operations can be performed:
+Just below the chart's preview, the following operations can be performed: 
 
 * Define the **X Attribute** of the chart (or **Group by** for *Pie Charts*) choosing between layer fields
 
 * Define the **Y Attribute** of the chart (or **Use** for *Pie Charts*) choosing between layer fields
 
-* Define the **Operation** with which the attributes will be related
+* Define the **Operation** with which the attributes will be related choosing between `No Operation`, `COUNT`, `SUM`, `AVG`, `STDDEV`, `MIN` and`MAX`
 
-* Choose the **Color** of the chart (or the **Color Ramp** for *Pie Charts*)
+!!! Note
+    The *No operation* option is used when you don't want to use the aggregation method
+
+* Choose the **Color** (`Blue`, `Red`, `Green`, `Brown` or `Purple`) of the chart (or the **Color Ramp** for *Pie Charts*)
+
+* Enable the **Display Legend**
+
+In addition, only for *Bar Charts* and *Line Charts*, [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) allows you to customize the chart by enabling the *Advanced Options* section.
 
-* Choose to keep the legend hidden, or to **Display Legend**
+<img src="../img/widgets/advanced_options.jpg" class="ms-docimage"/>
 
-In addition, only for *Bar Charts* and *Line Charts*, it is possible to access the *Advanced Options* section, where the user can:
+After that the user is allowed to:
 
-* Keep the grid visible or **Hide Grid**
+* Enable the **Hide Grid** of the chart
 
-* Keep Y axis visible or **Hide Y axis**
+* Customize **Y axis** data values by choosing the *Type* (`Auto`, `Linear`, `Category`, `Log` or `Date`); adding the *Prefix* (e.g. `~`), the *Format* (e.g. `.0%`:  multiply by 100 and suffix with `%`), the *Suffix* (e.g. `Km`) and the *Formula* to transform the value using a formula (e.g. `value + 2`) or hide the label enabling the *Hide labels*.
+
+<img src="../img/widgets/yaxis_options.jpg" class="ms-docimage"/>
+
+!!! Note
+    More information on the syntax options for the **Format** are available [here](https://d3-wiki.readthedocs.io/zh_CN/master/Formatting/) and for the **Formula** are available [here](https://github.com/m93a/filtrex#expressions). 
 
-* Define the **X Axis Labels rotation angle**
+* Customize **X axis** data values by choosing the *Type* (`Auto`, `Linear`, `Category`, `Log` or `Date`); hide the label enabling the *Hide labels* or enabling the *Never skip labels* and enabling the *Label rotation* to choose the rotation angle. 
```

Comment:
* Customize **X axis**  tick values by choosing the *Type* (between `Auto`, `Linear`, `Category`, `Log` or `Date`): the axis type is auto-detected by looking at data (*Auto* option is automatically managed and selected by the tool and it is usually good as default setting). As per **Y axis**, the user can completely hide labels through the *Hide labels* control or tune the rendering of tick labels with options like *Never skip labels* (it forces all ticks available in the chart to be rendered instead of simplifying the provided set based on chart size) and *Label rotation* to better adapt X axis tick labels on the charts depending on the needs.